docs/root/configuration/http/http_filters/_include/admission-control-filter.yaml

```diff
@@ -0,0 +1,72 @@
+static_resources:
+  listeners:
+    - address:
```

Comment:
one small nit: could you use "least" indent for the yaml lists 

we dont have ci currently to check this but it makes yaml config more readable i think

Reply:
Done. Didn't see a change with `yq -I1` but pyyaml seemed to do the trick.